assignments/number-functions/code.js

```diff
@@ -0,0 +1,30 @@
+const add = (a,b) => {
+  return a + b ;
+};
+const subtract = (a,b) => {
+  return a - b ;
+};
+const multiply = (a,b) => {
+  return a * b ;
+};
+const divide = (a,b) => {
+  return a / b ;
+};
+const mod = (a,b) => {
+  return a % b ;
+};
+const averageOfTwo = (a,b) => {
+  return (a+b)/2 ;
+};
+const averageOfThree = (a,b,c) => {
+  return (a+b+c)/3 ;
+};
+const distance = (a,b) => {
+  return Math.abs(a-b) ;
+};
+const manhattanDistance = (x1,y1,x2,y2) => {
+  return Math.abs(x1-x2)+Math.abs(y1-y2) ;
```

Comment:
This is perfectly fine but just to point out: you could make this arguably even more clear to the reader by using the `distance` function you just defined:

```suggestion
  return distance(x1, x2) + distance(y1, y2) ;
```

That makes it really clear even to someone who doesn't know what Manhattan distance is, that it's the sum of the distance on the x-axis and the distance on the y-axis without them having to mentally translate `Math.abs(x1 - x2)` back into "distance on the x-axis".